packages/plugin-coinbase/src/templates.ts

```diff
@@ -104,3 +104,99 @@ Provide the details in the following JSON format:
 Here are the recent user messages for context:
 {{recentMessages}}
 `;
+
+export const tokenContractTemplate = `
```

Comment:
Where is that template used? I don't see that anywhere

Reply:
Both are used in packages/plugin-coinbase/src/plugins/tokenContract.ts